doc/arkode/guide/source/Butcher.rst

```diff
@@ -331,6 +331,42 @@ This is the default 3th order slow and fast MRIStep method (from
    Linear stability region for the Knoth-Wolke method
 
 
+.. _Butcher.Sofroniou_Spaletta:
+
+Sofroniou-Spaletta-5-3-4
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: Sofroniou-Spaletta-5-3-4 ERK method
+
+Accessible via the constant ``ARKODE_SOFRONIOU_SPALETTA_5_3_4`` to
+:c:func:`ARKStepSetTableNum`, :c:func:`ERKStepSetTableNum`
+or :c:func:`ARKodeButcherTable_LoadERK`.
+Accessible via the string ``"ARKODE_SOFRONIOU_SPALETTA_5_3_4"`` to
+:c:func:`ARKStepSetTableName`, :c:func:`ERKStepSetTableName` or
+:c:func:`ARKodeButcherTable_LoadERKByName`.
+(from :cite:p:`SoSp:04`).
```

Comment:
Looks like this reference is missing from `doc/shared/sundials.bib` and might be why the CI doc build is failing

Reply:
Ah good catch